Core/include/Acts/TrackFinding/TrackSelector.hpp

```diff
@@ -342,6 +343,7 @@ void TrackSelector::selectTracks(const input_tracks_t& inputTracks,
 template <typename track_proxy_t>
 bool TrackSelector::isValidTrack(const track_proxy_t& track) const {
   auto checkMin = [](auto x, auto min) { return min <= x; };
+  auto checkMax = [](auto x, auto max) { return x <= max; };
```

Comment:
```suggestion
  auto checkMax = [](auto x, auto max) { return x < max; };
```
the usual definition of the range: min <= x < max.

Reply:
sorry, these are integers, so we want `<=` as you wrote. Please ignore my last comment (assuming I can't delete it)